### ISSUE TITLE
Question regarding vott release

### ISSUE THREAD

**Issue by kurmanadhvalla** [microsoft/VoTT#829]
Will there be any new release of vott in near future? If so,what will be the features we can expect?

**Comment by mienhuang** [comment]
for me, I also want to know. the recent time I am going to start a new project. if there is a new version of the plan, I am going to use the latest one.

**Comment by mloenow** [comment]
@wbreza @tbarlow12 @JacopoMangiavacchi @mydiemho @elizabethhalper @pjlittle @luisamiranda @tmiller-msft @PIC123 @golee ~ sorry for spamming you all, but could we please have some clarification here — has the project been abandoned? 🤔

**Comment by wbreza** [comment]
@mloenow, @kurmanadhvalla, @mienhuang   - The project has not been abandoned but right now we are not able to contribute at the same level as we did when launching the v2 version a few months back.

**Comment by mloenow** [comment]
@wbreza - thanks for the update. I'm a bit sad to hear that, as this was one of the better and more intuitive open source annotation tools I've come across (good work, really!). There's a lot of momentum in https://github.com/opencv/cvat, but it's not as user friendly IMO.

Could you at least start to look at some of the pull requests by the community and perhaps give some guidance on the active learning issues (no one seems to be able to get their custom models running - me included)?

I hope you'll be able to focus more in this project again in the future 🙏

**Comment by JacopoMangiavacchi** [comment]
@mloenow for the custom model loading a suggestion could be to use TensorFlow Object Detection API (https://github.com/tensorflow/models/tree/master/research/object_detection) to train your custom model and then use the TensorFlow.js Converter (https://github.com/tensorflow/tfjs-converter) to convert the TF model to the TF.js format needed by VoTT.

Once you have your model saved in the TF.js format put the model.json file, the weights binary files referenced in the model.json file and the custom classes.json file containing your labels on a folder and point to that folder from VoTT Active Learning settings.  Please look at the cocoSSDModel on this repo for a sample of all of these files.

**Comment by mloenow** [comment]
@JacopoMangiavacchi Thanks for jumping in here. That's exactly what I've tried to do, but for some reason it wouldn't load. The error message wasn't specific enough and I couldn't build from source at the time I tried to get it to work. Other people are having the same issue, e.g. https://github.com/microsoft/VoTT/issues/834 and https://github.com/microsoft/VoTT/issues/805.

I'll see if I can carve out some time to debug it again - however, it would be helpful to know exactly what kind of model configuration that was used in the included one (and/or provide a more generic support).

**Comment by JacopoMangiavacchi** [comment]
@mloenow the default model we shipped in VoTT is based on SSD/MobileNet architecture and it is trained with the COCO dataset.  You can find it on the tf.js model zoo (https://github.com/tensorflow/tfjs-models/tree/master/coco-ssd).

If you can share your model I can try to debug it too

**Comment by mloenow** [comment]
@JacopoMangiavacchi I believe I tried with a fine-tuned, quantized SSD MobileNet v2. I'll try to give it another try this week.

https://github.com/microsoft/VoTT/issues/846 (new) seems to be similar to my experience.

**Comment by mloenow** [comment]
@JacopoMangiavacchi I got it working following the steps in https://github.com/microsoft/VoTT/issues/846#issuecomment-520300055. Specifying the output node names seemed to make the difference. I'm seeing the same issue with the label at index 0 - luckily that was sorta like an unknown class in our case anyway, so removing it from classes.json worked. There's an open pull request that looks like it fixes this: https://github.com/microsoft/VoTT/pull/811 (needs to retarget `develop`). Perhaps we could have patch release with this and the other approved pull requests soon? 🙏🙂
